test/e2e/template/helmfile/testdata/snapshot/pr_560/output.yaml

```diff
@@ -1,103 +1,5 @@
-vals:
-map[]
-defaultVals:[]
-second-pass rendering result of "input.yaml.part.0":
- 0: releases:
- 1: - name: foo
- 2:   chart: ../../charts/raw-0.1.0
- 3:   values:
- 4:   - templates:
- 5:       - |
- 6:         apiVersion: v1
- 7:         kind: ConfigMap
- 8:         metadata:
- 9:           name: {{ .Release.Name }}-1
-10:           namespace: {{ .Release.Namespace }}
-11:         data:
-12:           foo: FOO
-13:   - git::https://github.com/helmfile/helmfile.git@test/e2e/template/helmfile/testdata/snapshot/pr_560/values.yaml?ref=main
-14:   secrets:
-15:   - git::https://github.com/helmfile/helmfile.git@test/e2e/template/helmfile/testdata/snapshot/pr_560/secrets.yaml?ref=main
-16:   missingFileHandler: Debug
-17: 
-
-merged environment: &{default map[] map[]}
-helm> $HelmVersion
-helm> 
 Building dependency release=foo, chart=../../charts/raw-0.1.0
-exec: helm dependency build ../../charts/raw-0.1.0
-1 release(s) found in input.yaml
-
-processing 1 groups of releases in this order:
-GROUP RELEASES
-1     foo
-
-processing releases in group 1/1: foo
-remote> getter: git
-remote> scheme: https
-remote> user: 
-remote> host: github.com
-remote> dir: /helmfile/helmfile.git
-remote> file: test/e2e/template/helmfile/testdata/snapshot/pr_560/values.yaml
-remote> home: /home/runner/.cache/helmfile
-remote> getter dest: values/https_github_com_helmfile_helmfile_git.ref=main
-remote> cached dir: /home/runner/.cache/helmfile/values/https_github_com_helmfile_helmfile_git.ref=main
-remote> downloading git::https://github.com/helmfile/helmfile.git?ref=main to values/https_github_com_helmfile_helmfile_git.ref=main
-client: {Ctx:context.Background Src:git::https://github.com/helmfile/helmfile.git?ref=main Dst:/home/runner/.cache/helmfile/values/https_github_com_helmfile_helmfile_git.ref=main Pwd:/home/runner/.cache/helmfile Mode:3 Umask:---------- Detectors:[] Decompressors:map[] Getters:map[] Dir:false ProgressListener:<nil> Insecure:false DisableSymlinks:false Options:[]}
-skipping missing values file matching "git::https://github.com/helmfile/helmfile.git@test/e2e/template/helmfile/testdata/snapshot/pr_560/values.yaml?ref=main"
-remote> getter: git
-remote> scheme: https
-remote> user: 
-remote> host: github.com
-remote> dir: /helmfile/helmfile.git
-remote> file: test/e2e/template/helmfile/testdata/snapshot/pr_560/secrets.yaml
-remote> home: /home/runner/.cache/helmfile
-remote> getter dest: values/https_github_com_helmfile_helmfile_git.ref=main
-remote> cached dir: /home/runner/.cache/helmfile/values/https_github_com_helmfile_helmfile_git.ref=main
-skipping missing secrets file matching "git::https://github.com/helmfile/helmfile.git@test/e2e/template/helmfile/testdata/snapshot/pr_560/secrets.yaml?ref=main"
 Templating release=foo, chart=../../charts/raw-0.1.0
-exec: helm template foo ../../charts/raw-0.1.0 --values /tmp/helmfile/foo-values-d459bc67c --debug
-helm> install.go:192: [debug] Original chart version: ""
```

Comment:
I had to disable debug logging for this test case because running it with yaml.v2 resulted in adding an extra `helm>` line here due to an unknown reason.
It looked harmless but hard to solve so I just disabled the debug log to make it stable.